ui/ducks/send/send.js

```diff
@@ -2378,12 +2383,20 @@ export function signTransaction() {
         ),
       );
 
-      dispatch(
+      const result = await dispatch(
         addTransactionAndRouteToConfirmationPage(txParams, {
           sendFlowHistory: draftTransaction.history,
           type: transactionType,
         }),
       );
+      dispatch({
+        type: SET_DETAILS_FOR_CONFIRM_TX,
```

Comment:
Rather than doing a raw dispatch, could we add a helper method to `confirm-transaction.duck.js` as is done with some of the existing actions?